src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -6,39 +6,26 @@
 from ._util import (get_sql_servers_operation, get_sql_database_operations,
                     get_sql_elasticpools_operations, get_sql_recommended_elastic_pools_operations,
                     ServiceGroup, create_service_adapter)
+from azure.cli.core.commands import cli_command
 
-server_operations = create_service_adapter('azure.mgmt.sql.operations.servers_operations',
-                                           'ServersOperations')
-
-with ServiceGroup(__name__, get_sql_servers_operation, server_operations) as s:
-    with s.group('sql server') as c:
-        c.command('create', 'create_or_update')
-        c.command('delete', 'delete')
-        c.command('show', 'get_by_resource_group')
-        c.command('show-usage', 'list_usages')
-        c.command('list', 'list_by_resource_group')
-        c.generic_update_command('update', 'get_by_resource_group', 'create_or_update')
-
-    with s.group('sql server service-objective') as c:
-        c.command('list', 'list_service_objectives')
-        c.command('show', 'get_service_objective')
-
-    with s.group('sql server firewall') as c:
-        c.command('create', 'create_or_update_firewall_rule')
-        c.command('update', 'create_or_update_firewall_rule')
-        c.command('delete', 'delete_firewall_rule')
-        c.command('show', 'get_firewall_rule')
-        c.command('list', 'list_firewall_rules')
+###############################################
+#                sql db                       #
+###############################################
 
 database_operations = create_service_adapter('azure.mgmt.sql.operations.databases_operations',
                                              'DatabasesOperations')
 
 with ServiceGroup(__name__, get_sql_database_operations, database_operations) as s:
     with s.group('sql db') as c:
-        c.command('create', 'create_or_update')
+        c.custom_command('create', 'db_create')
+        c.custom_command('copy', 'db_copy')
+        c.custom_command('create-secondary', 'db_create_secondary')
+        ## Commented out for now because restorePointInTime property is not yet in Swagger spec
+        #c.custom_command('restore', 'db_restore')
         c.command('show', 'get')
-        c.command('list', 'list_by_server')
-        c.command('show-usage', 'list_usages')
+        c.custom_command('list', 'db_list')
+        ## Usages will not be included in the first batch of GA commands
+        #c.command('show-usage', 'list_usages')
         c.command('delete', 'delete')
         c.generic_update_command('update', 'get', 'create_or_update')
```

Comment:
Additionally if the command name is "update" should we first validate that the database exists rather than creating it?